test/circle/yaml-schemas/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+name: str()
+home: str()
+version: str()
+appVersion: any(str(), num())
+description: str()
+keywords: list(str(), required=False)
+sources: list(str(), required=False)
+maintainers: list(include('maintainer'))
+icon: str(required=False)
+---
+maintainer:
+  name: str()
+  email: str(required=False)
```

Comment:
There are more than just these that can be in a chart. The options are described in [a proto file](https://github.com/kubernetes/helm/blob/master/_proto/hapi/chart/metadata.proto). Could we get more of the properties added to the schema?